NitroxClient/Unity/Helper/DebugUtils.cs

```diff
@@ -7,6 +7,46 @@ namespace NitroxClient.Unity.Helper
 {
     public class DebugUtils
     {
+        private static string prefix = "";
+
+        public static void DumpHierarchy(GameObject obj)
+        {
+            Log.Debug("Analyze gameobject: ");
+            Log.Debug("------------------------------------");
+            StartHierachy(obj);
+            Log.Debug("------------------------------------");
+        }
+
+        private static void StartHierachy(GameObject obj)
+        {
+            prefix = prefix + "    ";
+            Log.Debug(GetInfoOfGamobject(obj));
+            for (int i = 0; i < obj.transform.childCount; i++)
```

Comment:
Why the indices? The other `DumpGameObject` example clearly shows that you can do this in tidy `foreach` loop. If `Transform` were using a linked list, `GetChild(i)` would be immensely slow.